mim/utils/utils.py

```diff
@@ -44,6 +46,56 @@ def parse_url(url: str) -> Tuple[str, str]:
     return username, repo
 
 
+def is_installed(package: str) -> bool:
+    """Check package whether installed.
+
+    Args:
+        package (str): Name of package to be checked.
+    """
+    # refresh the pkg_resources
+    # more datails at https://github.com/pypa/setuptools/issues/373
+    importlib.reload(pkg_resources)
+    try:
+        get_distribution(package)
+        return True
+    except pkg_resources.DistributionNotFound:
+        return False
+
+
+def check_installation_decorator(func):
```

Comment:
check_installation_decorator -> check_installed

Reply:
got it